code/socialDistribution/static/author/style.css

```diff
@@ -3,13 +3,38 @@
     margin: 0 auto; 
 }
 
+.author-prof {
+    max-width: 50rem;
+    height: 10rem;
+    margin: 0 auto;
+}
+
+.author-tile-link {
+    text-decoration: none;
+    color: inherit;
+    outline: none;
+}
+
 .author-tile {
     border-radius: 30px;
 }
 
+.author-tile:hover {
+    border-color: lightblue !important;
```

Comment:
Kinda picky, but I'm pretty sure this is a bad practice lol. It's fine though.

Reply:
Lol yes, sorry, it was so hard to get the border working because of bootstrap. I'll try to fix it though

Reply:
Was able to fix. There was a lingering border-white after the refactoring.